src/wallets/coinbase/actions.ts

```diff
@@ -189,13 +189,15 @@ export const createAccount = (page: Page) => async (): Promise<void> => {
   } catch {
     // Ignore missing help prompt
   }
+
+  await waitForChromeState(page);
 };
 
 export const switchAccount =
   (page: Page) =>
   async (i: number): Promise<void> => {
     await page.getByTestId('portfolio-header--switcher-cell-pressable').click();
-    (
+    await (
```

Comment:
May also need to `await` on line 120 when selecting the first item in the network list

Reply:
oooo true. nice find.